test/src/runner/testRunnerEs6Async.js

```diff
@@ -196,7 +196,7 @@ describe('testRunner ES6 Async', function() {
           'element-6066-11e4-a52e-4f735466cecf': '5cc459b8-36a8-3042-8b4a-258883ea642b'
         }]
       })
-    }, true, true);
+    }, false, true);
```

Comment:
is this needed?

Reply:
Oh I see, findElement also uses the /elements endpoint, not the /element. Nevermind then.